main.css

```diff
@@ -602,3 +602,225 @@ ul {
   width: 10px;
   font-weight: 600;
 }
+
+/* Duplicated from line: 366
+ *
+.project__button {
+  display: flex;
+  flex-direction: row;
+  padding: 12px;
+  gap: 10px;
+  margin-top: 10px;
+  border: 1px solid #6070ff;
+  width: 126px;
+  height: 48px;
+  position: absolute;
+  border-radius: 8px;
+  cursor: pointer;
+  color: #4657f5;
+  font-size: 16px;
+  background-color: rgba(235, 24, 24, 0.945);
+  animation-name: portfolio;
+  animation-duration: 10s;
+}
+* */
+
+.myself {
+  background-size: cover;
+  width: 100%;
+  height: 1263px;
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+  background-color: #fff;
+  margin-top: 114px;
+  border-top-right-radius: 80px;
+}
+
+/*  Duplicated from line: 515
+ *
+.myself h4 {
+  width: 327px;
+  height: 35px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1f3863;
+  margin-top: 114px;
+}
+* */
+
+.contact__container_me h2 {
+  text-align: center;
+  margin-bottom: var(--bottom-margin-2);
+}
+
+/* Duplicated from line: 529
+ *
+.myself > text > p {
+  width: 100%;
+  height: 145px;
+  color: #344563;
+  font-family: Poppins, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 24px;
+  margin-top: 12px;
+  order: 1;
+}
+* */
+
+/* Duplicated from line: 550
+ *
+.resume__button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  background-color: #fff;
+  margin-top: 22px;
+  border-radius: 8px;
+  border: 1px solid #5f6ef1;
+  width: 130px;
+  height: 40px;
+}
+* */
+
+/* Duplicated from line: 569
+ *
+.resume__coding {
+  display: flex;
+  margin-top: 36px;
+  background-color: #f6f6ff;
+  width: 303px;
+  height: 66px;
+  margin-left: 16px;
+  border-radius: 8px;
+}
+* */
+
+/* Duplicated from line: 589
+.resume__coding img {
+  display: flex;
+  margin-top: 8px;
+  margin-left: 10px;
+  width: 40px;
+  height: 38px;
+}
+* */
+
+/* Duplicated from line: 597
+ *
+.framework__container {
+  display: flex;
+  margin-top: 70px;
+  color: #172b4d;
+  height: 20px;
+  width: 10px;
+  font-weight: 600;
+}
+* */
```

Comment:
:red_circle: {REQUIRED] I noticed that you have some commented code in your code. Commented code is called dead code because it is not executed. It is important to avoid committing dead code to GitHub because it can make your code more difficult to understand and maintain.

I recommend that you remove all of the commented code from your code before you push it to GitHub. This will make your code cleaner and easier to understand, maintain, and perform.